public/Player.js

```diff
@@ -1,16 +1,19 @@
 "use strict";
 //this class is going to create the players
+import PlayHand from "./PlayHand.js";
 
 export default class Player {
   domElement = null;
 
-  constructor(name, pot = 200, playerType = "GuestPlayer") {
+  constructor(confingObject) {
```

Comment:
There is a typo:
```suggestion
  constructor(configObject) {
```
Also a better name would be `playerConfig` as it can be assumed it will be in object